docs/components/receive_route.md

```diff
@@ -0,0 +1,119 @@
+
+Flags:
+  -h, --help                  Show context-sensitive help (also try --help-long
+                              and --help-man).
+      --version               Show application version.
+      --log.level=info        Log filtering level.
+      --log.format=logfmt     Log format to use. Possible options: logfmt or
+                              json.
+      --tracing.config-file=<file-path>
+                              Path to YAML file with tracing configuration. See
+                              format details:
+                              https://thanos.io/tip/thanos/tracing.md/#configuration
+      --tracing.config=<content>
+                              Alternative to 'tracing.config-file' flag (lower
+                              priority). Content of YAML file with tracing
+                              configuration. See format details:
+                              https://thanos.io/tip/thanos/tracing.md/#configuration
+      --http-address="0.0.0.0:10902"
+                              Listen host:port for HTTP endpoints.
+      --http-grace-period=2m  Time to wait after an interrupt received for HTTP
+                              Server.
+      --remote-write.address="0.0.0.0:19291"
+                              Address to listen on for remote write requests.
+      --remote-write.server-tls-cert=""
+                              TLS Certificate for HTTP server, leave blank to
+                              disable TLS.
+      --remote-write.server-tls-key=""
+                              TLS Key for the HTTP server, leave blank to
+                              disable TLS.
+      --remote-write.server-tls-client-ca=""
+                              TLS CA to verify clients against. If no client CA
+                              is specified, there is no client verification on
+                              server side. (tls.NoClientCert)
+      --remote-write.client-tls-cert=""
+                              TLS Certificates to use to identify this client to
+                              the server.
+      --remote-write.client-tls-key=""
+                              TLS Key for the client's certificate.
+      --remote-write.client-tls-ca=""
+                              TLS CA Certificates to use to verify servers.
+      --remote-write.client-server-name=""
+                              Server name to verify the hostname on the returned
+                              gRPC certificates. See
+                              https://tools.ietf.org/html/rfc4366#section-3.1
+      --receive.hashrings-file=<path>
+                              Path to file that contains the hashring
+                              configuration.
+      --receive.hashrings-file-refresh-interval=5m
+                              Refresh interval to re-read the hashring
+                              configuration file. (used as a fallback)
+      --receive.local-endpoint=RECEIVE.LOCAL-ENDPOINT
```

Comment:
This is not related to this PR but a bit related to your work with the router.  To me, this flag doesn't make any sense after the split. Is this flag actually used in the code?